models/articles.model.js

```diff
@@ -12,4 +12,30 @@ function fetchArticlesById(id) {
     })
 }
 
-module.exports = fetchArticlesById
\ No newline at end of file
+async function fetchArticles() {
+    
+        const articlesData = await db.query(
+            `SELECT * FROM articles ORDER BY created_at DESC`
+        )
+        const commentsData = await db.query(
```

Comment:
is there one query that we could do that would fetch all this information for us? Hint, we are using 'relational' databases and we have the JOIN keyword we can use :D